dev-lang/python/python3.10-3.10.8.recipe

```diff
@@ -11,28 +11,28 @@ LICENSE="Python"
 COPYRIGHT="1990-2022 Python Software Foundation"
 REVISION="1"
 SOURCE_URI="https://www.python.org/ftp/python/$portVersion/Python-$portVersion.tar.xz"
-CHECKSUM_SHA256="6eed8415b7516fb2f260906db5d48dd4c06acc0cb24a7d6cc15296a604dcdc48"
+CHECKSUM_SHA256="6a30ecde59c47048013eb5a658c9b5dec277203d2793667f578df7671f7f03f3"
 SOURCE_DIR="Python-$portVersion"
-PATCHES="python310-$portVersion.patchset"
+PATCHES="python3.10-$portVersion.patchset"
 if [ "$secondaryArchSuffix" = _x86 ] ; then
 	PATCHES+="
-		python310_x86-$portVersion.patchset
+		python3.10_x86-$portVersion.patchset
 	"
 fi
 
 ARCHITECTURES="all !x86_gcc2 ?x86"
 SECONDARY_ARCHITECTURES="x86"
 
 # On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
-commandSuffix=$secondaryArchSuffix
 commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
 if [ "$targetArchitecture" = x86_gcc2 ]; then
-	commandSuffix=
 	commandBinDir=$prefix/bin
+	commandSuffix=
 fi
 
 PROVIDES="
-	python310$secondaryArchSuffix = $portVersion compat >= 3.10
```

Comment:
I think you need to keep this one (see the python3.9 recipe), this is picked up for recipes depending on `PYTHON_PACKAGES=(python3 python38 python39 python310)`.

Reply:
> I was assuming that was what the REPLACES was for.

I think in this case there isn't anything to replace as there wouldn't be anything providing python310?

> Is there any documentation/guidelines about how to handle situations like this?

No documentation, but as in the case with `-` vs `_` in recipes, here you can't use dot's in `PYTHON_PACKAGES` :)